### DISCUSSION TITLE
TypeError: __init__() got an unexpected keyword argument 'log_every_n_steps'

### DISCUSSION THREAD

**Discussion by LilianaA1995** [Lightning-AI/pytorch-lightning#13247] [General]
I was trying to run this code for LabelMe data through CNN :

```python
trainer = pl.Trainer(min_epochs=5,
                     max_epochs=30,
                     callbacks=[EarlyStopping(monitor='val_loss', patience=3, mode='min')],
                     log_every_n_steps=2)

trainer.fit(model)
```

but this error has appeared.. __init__() got an unexpected keyword argument 'log_every_n_steps'. 

How to solve it ?

My skills in coding are really poor, so please, understand.

**Comment by akihironitta** [comment]
@LilianaA1995 You might have a very old version of PyTorch Lightning installed in your environment. Could you check the version you're using by running either of the following commands?

```console
$ python -c "import pytorch_lightning as pl; print(pl.__version__)
$ python -m pip list
```
